content/en/blog/2024/ambient-reaches-beta/index.md

```diff
@@ -1,81 +1,86 @@
 ---
-title: "Istio’s new sidecar-less ‘Ambient Mode’ reaches Beta in version 1.22"
-description: The latest release from Istio brings service mesh Layer 4 & 7 features to production readiness without sidecars.
+title: "Say goodbye to your sidecars: Istio's ambient mode reaches Beta in v1.22"
+description: Layer 4 & Layer 7 features are now ready for wide adoption.
```

Comment:
I think i like the original desp better. It is important to highlight mesh l4 & l7 features & production readiness.

Reply:
The only reason we say "L4 and L7 features" is that we were saying for a while it would _only_ have been L4 features.  I also prefer not to say "production readiness" without "without precautions/caveats" so I'm torn.  The real win was not to mention "sidecarless" in both the heading and the subheading